### PR TITLE
GETOK WebADE removal and Node Update

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2319
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2292
Remove old v1 app code backup
Remove all webade references 
Change alpine image
Big package update
Node 16 changes

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

You can ignore the deleted files (under /backend and /frontend roots), they're the back-up of the old codebase we were keeping around

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://getok-dev.apps.silver.devops.gov.bc.ca/pr-318

**Review by jujaga — APPROVED** [review]

File: openshift/app.dc.yaml [hunk 1]
```diff
@@ -216,36 +216,6 @@ objects:
...
-                - name: SC_WEBADE_PROD_USERNAME
-                  valueFrom:
-                    secretKeyRef:
-                      key: username
-                      name: getok-sc-webade-prod-secret
-                - name: SC_WEBADE_PROD_PASSWORD
-                  valueFrom:
-                    secretKeyRef:
-                      key: password
-                      name: getok-sc-webade-prod-secret
```
Comment (TimCsaky) [inline openshift/app.dc.yaml:248]: i'm guessing this will remove the webade secrets in openshift that we will no longer need

Comment (loneil) [reply openshift/app.dc.yaml:248]: Will remove them from the deployment but they stay in the secrets in the namespace(s).
Will clean up appropriate configs that aren't needed any more, but the **passwords for the webade clients** will need to remain, as those are existing clients for this team in the webade system that could be referenced again if needed in the future for any reason.

Comment (TimCsaky) [reply openshift/app.dc.yaml:248]: ok. got it. thanks

**Review by TimCsaky — APPROVED** [review]
App installed, ran, tested all fine for me (on node 16)
Yeah, everything just worked. 
I guess people can checkout earlier commits if they want to review how the WebADE stuff was done.

**Pull request merged**